### DISCUSSION TITLE
直接插入Bitmap数据

### DISCUSSION THREAD

**Discussion by doingf** [StarRocks/starrocks#7901] [Announcements]
看文档描述BitMap类型的数据写入都需要通过to_bitmap()函数的定义转换，如果我想通过代码直接load进去bitmap类型的值, 不经过函数转换；使用Java代码请问应该怎么操作?测试通过Jdbc的setBytes()或setObject()都无法实现（借用了BitmapValue.java代码逻辑）

**Comment by alberttwong** [comment]
Sorry, can you ask chinese based questions on wechat.
